### PR TITLE
Ndt/no cache

### PR DESCRIPTION
**Pull request by ndormont**
Added no cache option to build process, for the docker tasks.

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/gradle-docker](https://github.com/palantir/gradle-docker), @ndormont! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy [hunk 1]
```diff
@@ -93,6 +93,10 @@ class PalantirDockerPlugin implements Plugin<Project> {
             }
 
             List<String> buildCommandLine = ['docker', 'build']
+            if (ext.noCache) {
+                buildCommandLine.add '--no-cache'
+                log.info("No cache option ON")
```
Comment (uschi2000) [inline src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy]: don't think we need this; if anything, we can log the final docker call

Comment (ndormont) [reply src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy]: just to be clear, you'd like me to remove the log.info ? thx

**Review by uschi2000 — APPROVED** [review]
sorry, missed this in my inbox.

**Pull request merged**